app/controllers/map_controller.rb

```diff
@@ -16,7 +16,10 @@ def index
   end
 
   def show
-    @node = DrupalNode.find_map_by_slug(params[:name]+'/'+params[:date])
+    @node = DrupalNode.find params[:name]
```

Comment:
Yeah we can always use an url like `/map/NewYork-map-07-22-2016/07-22-2016` for maps (where `NewYork-map-07-22-2016` is the slug) and something similar to that in notes. Only thing is that it looks like overuse of the date in the slugs.